SagaFlow.AspNetCore/MvcProvider/SchemaController.cs

```diff
@@ -76,9 +77,10 @@ public class ResourceMetadata
 
     public class CommandDefinition
     {
-        public string Name { get; set; }
-        public string Href { get; set; }
-        public IDictionary<string,ParameterDefinition> Parameters { get; set; }
+        public string Name { get; init; }
+        public string Description { get; init; }
+        public string Href { get; init; }
```

Comment:
Should a command always have a href or should that be part of it's meta data/parameters?

Reply:
if we were following HAL or HATEOAS, then it should really live in a _links property in the returning model.  But I think what Tyson has done is fine for the moment.